src/broadcast-bot.js

```diff
@@ -281,7 +281,7 @@ async function listen(message) {
 
     if (webAppEnabled) {
       webAppString = '*Web App Commands*\n'
-        + '/panel : displays the link and token to the web user interface';
+        + '/panel : displays the link and token to the web user interface\n\n';
```

Comment:
If there is no web app, then you need another carriage return
If (!webAppEnabed) {
  webAppString = '\n'
}